keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py

```diff
@@ -353,7 +354,7 @@ def __init__(
                 first_shortcut=(block_type == "block" or stack_index > 0),
                 stack_index=stack_index,
             )
-            stack_level_outputs[stack_index + 2] = x
+            stack_level_outputs[stack_index + 2] = output_name
```

Comment:
naming: "stack" does not mean much outside of this implementation
"pyramid_levels" ?
"pyramid_level_outputs" ?
They are called "Pn" in literature for a reason.

Reply:
+1